gmp/gmp.py

```diff
@@ -1269,10 +1271,125 @@ def modifyTagCommand(self, tag_id, kwargs):
                ''.format(tag_id, name, resource, value, comment, active)
 
     def modifyTargetCommand(self, target_id, kwargs):
-        raise NotImplementedError
+        
+        if not target_id:
+            raise ValueError('modify_target requires a target_id element')
 
-    def modifyTaskCommand(self):
-        raise NotImplementedError
+        comment = kwargs.get('comment', '')
+        if comment:
+            comment = '<comment>%s</comment>' % comment
+
+        name = kwargs.get('name', '')
+        if name:
+            name = '<name>%s</name>' % name
+
+        hosts = kwargs.get('hosts', '')
+        if hosts:
+            hosts = '<hosts>%s</hosts>' % hosts
+
+        comment = kwargs.get('comment', '')
+        if comment:
+            comment = '<comment>%s</comment>' % comment
+
+        copy = kwargs.get('copy', '')
+        if copy:
+            copy = '<copy>%s</copy>' % copy
+
+        exclude_hosts = kwargs.get('exclude_hosts', '')
+        if exclude_hosts:
+            exclude_hosts = '<exclude_hosts>%s</exclude_hosts>' % exclude_hosts
+
+        alive_tests = kwargs.get('alive_tests', '')
+        if alive_tests:
+            alive_tests = '<alive_tests>%s</alive_tests>' % alive_tests
+
+        reverse_lookup_only = kwargs.get('reverse_lookup_only', '')
+        if reverse_lookup_only:
+            reverse_lookup_only = '<reverse_lookup_only>%s</reverse_lookup_only>' % reverse_lookup_only
+        
+        reverse_lookup_unify = kwargs.get('reverse_lookup_unify', '')
+        if reverse_lookup_unify:
+            reverse_lookup_unify = '<reverse_lookup_unify>%s</reverse_lookup_unify>' % reverse_lookup_unify
+
+        port_range = kwargs.get('port_range', '')
+        if port_range:
+            port_range = '<port_range>%s</port_range>' % port_range
+
+        if 'port_list' in kwargs:
+            port_list = kwargs.get('port_list')
+            if 'id' in port_list:
+                port_list = '<port_list id="%s"/>' % port_list
+
+        return '<modify_target target_id="{0}">{1}{2}{3}{4}{5}{6}  \
```

Comment:
The `format` statement contains 12 placeholders in the string, but there are only 11 parameters passed to the method.

There is either one placeholder too many or a missing parameter.